### DISCUSSION TITLE
Developing inside docker container

### DISCUSSION THREAD

**Discussion by Jason-CKY** [directus/directus#7527] [Archive]
Is there a way to edit some of the source code inside the docker container?
I cannot find the /directus/app/src folders in the docker image on docker hub, only the following folders are seen in the working directory.
![image](https://user-images.githubusercontent.com/27609953/130255184-7a497631-9733-4e97-be36-c5eadb5c6e39.png)

May I know if there is a way to edit the source code and rebuild the image or to view the source code from within the directus container? 

An example would be to build an image with the Dockerfile below:

```dockerfile
FROM directus/directus:9.0.0-rc.90 as base
# edit the app.vue file and change it via the Dockerfile
COPY app.vue /directus/app/src/app.vue
```

**Comment by rijkvanzanten** [comment]
The bundling isn't done in the image on run time, but ahead of time when the release is made. You can't override any of the files in the source code like this.
